### ISSUE TITLE
{"ERROR": "Invalid path or could not find device!"}

### ISSUE THREAD

**Issue by Dieskz** [nythepegasus/SideJITServer#155]
I restarted my PC. And my iPad.The server has also been restarted.I checked my intranet IP.And UDID. None of them is wrong. None of these measures are effective.
![IMG_0027](https://github.com/user-attachments/assets/e6f35a5c-3a67-4575-b9f3-62083d5bf4e0)

**Comment by aaaaaaneuseok** [comment]
Just go to this site and try it.(your computer)127.0.0.1:8080/re

**Comment by Dieskz** [comment]
> 只需访问此站点并尝试一下。（你的电脑）127.0.0.1：8080/re
I don't understand.Can you be more specfic

**Comment by 10XS** [comment]
我也有同样的问题

**Comment by aaaaaaneuseok** [comment]
> > 이 사이트에 접속해서 시도해 보세요. ( 너의 컴퓨터)127.0.0.1:8080/re이해가 안 돼요. 좀 더 구체적으로 할 수 있나요?
enter site using your browser(opening your server)
like chrome or Firefox

http://127.0.0.1:8080/re

**Comment by 10XS** [comment]
What does it mean?





发自我的iPad


------------------ Original ------------------
From: aaaaaaneuseok ***@***.***&gt;
Date: Sun,Oct 27,2024 4:27 PM
To: nythepegasus/SideJITServer ***@***.***&gt;
Cc: 10XS ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [nythepegasus/SideJITServer] {"ERROR": "Invalid path or could notfind device!"} (Issue #155)





   
이 사이트에 접속해서 시도해 보세요. ( 너의 컴퓨터)127.0.0.1:8080/re이해가 안 돼요. 좀 더 구체적으로 할 수 있나요?
 enter site using your browser(opening your server)
 like chrome or Firefox
   
http://127.0.0.1:8080/re

**Comment by aaaaaaneuseok** [comment]
enter the site

**Comment by 10XS** [comment]
It can't be opened at all. &nbsp; 根本就打不开



发自我的iPad


------------------ Original ------------------
From: aaaaaaneuseok ***@***.***&gt;
Date: Sun,Oct 27,2024 4:33 PM
To: nythepegasus/SideJITServer ***@***.***&gt;
Cc: 10XS ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [nythepegasus/SideJITServer] {"ERROR": "Invalid path or could notfind device!"} (Issue #155)





 
enter the site

**Comment by aaaaaaneuseok** [comment]
Your computer

**Comment by 10XS** [comment]
什么意思？



发自我的iPad


------------------ Original ------------------
From: aaaaaaneuseok ***@***.***&gt;
Date: Sun,Oct 27,2024 4:36 PM
To: nythepegasus/SideJITServer ***@***.***&gt;
Cc: 10XS ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [nythepegasus/SideJITServer] {"ERROR": "Invalid path or could notfind device!"} (Issue #155)





 
Your computer

**Comment by 10XS** [comment]
What do you mean? Can you say more?



发自我的iPad


------------------ Original ------------------
From: aaaaaaneuseok ***@***.***&gt;
Date: Sun,Oct 27,2024 4:36 PM
To: nythepegasus/SideJITServer ***@***.***&gt;
Cc: 10XS ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [nythepegasus/SideJITServer] {"ERROR": "Invalid path or could notfind device!"} (Issue #155)





 
Your computer

**Comment by aaaaaaneuseok** [comment]
Enter the link using your computer

**Comment by 10XS** [comment]
I flipped through all the information on the Internet, and finally saw someone saying that they seemed to need a driver.





发自我的iPad


------------------ Original ------------------
From: aaaaaaneuseok ***@***.***&gt;
Date: Sun,Oct 27,2024 4:36 PM
To: nythepegasus/SideJITServer ***@***.***&gt;
Cc: 10XS ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [nythepegasus/SideJITServer] {"ERROR": "Invalid path or could notfind device!"} (Issue #155)





 
Your computer

**Comment by 10XS** [comment]
Browser or where? Can he help me solve the problem of error reporting? Thank you very much for your patience.



发自我的iPad


------------------ Original ------------------
From: aaaaaaneuseok ***@***.***&gt;
Date: Sun,Oct 27,2024 4:38 PM
To: nythepegasus/SideJITServer ***@***.***&gt;
Cc: 10XS ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [nythepegasus/SideJITServer] {"ERROR": "Invalid path or could notfind device!"} (Issue #155)





 
Enter the link using your computer

**Comment by aaaaaaneuseok** [comment]
browser

**Comment by 10XS** [comment]
You really can't blame me.





发自我的iPad


------------------ Original ------------------
From: aaaaaaneuseok ***@***.***&gt;
Date: Sun,Oct 27,2024 4:52 PM
To: nythepegasus/SideJITServer ***@***.***&gt;
Cc: 10XS ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [nythepegasus/SideJITServer] {"ERROR": "Invalid path or could notfind device!"} (Issue #155)





 
browser

**Comment by mdtoufique** [comment]
Try this method:
in my case,  it showed when i tried wirelessly. I solved it by disabling windows firewall and refreshing the server every time. I dont know why refreshing is needed every time i open it but it works  for me
After u start the server go to ur browser and type the full address like "htttp://192.168.0.104:8080/re" ur address maybe different,  just add "/re" at the end of ur server address

**Comment by 10XS** [comment]
Thank you very much. I will try it.



发自我的iPad


------------------ Original ------------------
From: MD Toufique ***@***.***&gt;
Date: Sun,Oct 27,2024 10:53 PM
To: nythepegasus/SideJITServer ***@***.***&gt;
Cc: 10XS ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [nythepegasus/SideJITServer] {"ERROR": "Invalid path or could notfind device!"} (Issue #155)





 
Try this method:
 in my case,  it showed when i tried wirelessly. I solved it by disabling windows firewall and refreshing the server every time. I dont know why refreshing is needed every time i open it but it works  for me
 After u start the server go to ur browser and type the full address like "htttp://192.168.0.104:8080/re" ur address maybe different,  just add "/re" at the end of ur server address

**Comment by 10XS** [comment]
It's successful!!! I sincerely thank you like you!



发自我的iPad


------------------ Original ------------------
From: MD Toufique ***@***.***&gt;
Date: Sun,Oct 27,2024 10:53 PM
To: nythepegasus/SideJITServer ***@***.***&gt;
Cc: 10XS ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [nythepegasus/SideJITServer] {"ERROR": "Invalid path or could notfind device!"} (Issue #155)





 
Try this method:
 in my case,  it showed when i tried wirelessly. I solved it by disabling windows firewall and refreshing the server every time. I dont know why refreshing is needed every time i open it but it works  for me
 After u start the server go to ur browser and type the full address like "htttp://192.168.0.104:8080/re" ur address maybe different,  just add "/re" at the end of ur server address

**Comment by 10XS** [comment]
I'm a little curious about how you came up with this method.







------------------ Original ------------------
From: MD Toufique ***@***.***&gt;
Date: Sun,Oct 27,2024 10:53 PM
To: nythepegasus/SideJITServer ***@***.***&gt;
Cc: 10XS ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [nythepegasus/SideJITServer] {"ERROR": "Invalid path or could notfind device!"} (Issue #155)





 
Try this method:
 in my case,  it showed when i tried wirelessly. I solved it by disabling windows firewall and refreshing the server every time. I dont know why refreshing is needed every time i open it but it works  for me
 After u start the server go to ur browser and type the full address like "htttp://192.168.0.104:8080/re" ur address maybe different,  just add "/re" at the end of ur server address

**Comment by aaaaaaneuseok** [comment]
Yes

**Comment by aaaaaaneuseok** [comment]
Actually,I'm not good at English

**Comment by 10XS** [comment]
I add /re at the end of the website and open it in the browser. But he prompted {"OK": "Refreshed!"}



发自我的iPad


------------------ Original ------------------
From: aaaaaaneuseok ***@***.***&gt;
Date: Tue,Oct 29,2024 9:59 PM
To: nythepegasus/SideJITServer ***@***.***&gt;
Cc: 10XS ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [nythepegasus/SideJITServer] {"ERROR": "Invalid path or could notfind device!"} (Issue #155)





 
Yes

**Comment by 10XS** [comment]
To tell you the truth, I actually use a translator.



发自我的iPad


------------------ Original ------------------
From: aaaaaaneuseok ***@***.***&gt;
Date: Tue,Oct 29,2024 10:03 PM
To: nythepegasus/SideJITServer ***@***.***&gt;
Cc: 10XS ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [nythepegasus/SideJITServer] {"ERROR": "Invalid path or could notfind device!"} (Issue #155)





 
It's refreshing server

**Comment by 10XS** [comment]
什么刷新服务器？



发自我的iPad


------------------ Original ------------------
From: aaaaaaneuseok ***@***.***&gt;
Date: Tue,Oct 29,2024 10:03 PM
To: nythepegasus/SideJITServer ***@***.***&gt;
Cc: 10XS ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [nythepegasus/SideJITServer] {"ERROR": "Invalid path or could notfind device!"} (Issue #155)





 
It's refreshing server

**Comment by 10XS** [comment]
Where are you?



发自我的iPad


------------------ Original ------------------
From: aaaaaaneuseok ***@***.***&gt;
Date: Tue,Oct 29,2024 10:03 PM
To: nythepegasus/SideJITServer ***@***.***&gt;
Cc: 10XS ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [nythepegasus/SideJITServer] {"ERROR": "Invalid path or could notfind device!"} (Issue #155)





 
It's refreshing server

**Comment by 10XS** [comment]
Excuse me, what is your native language?



发自我的iPad


------------------ Original ------------------
From: aaaaaaneuseok ***@***.***&gt;
Date: Tue,Oct 29,2024 10:03 PM
To: nythepegasus/SideJITServer ***@***.***&gt;
Cc: 10XS ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [nythepegasus/SideJITServer] {"ERROR": "Invalid path or could notfind device!"} (Issue #155)





 
It's refreshing server

**Comment by 10XS** [comment]
This is a very huge bug, involving a wide range. It seems that some sub-versions of ios17 will appear {"ERROR": "Could not find any device!"} I hope the author can fix the problem as soon as possible.

**Comment by Dieskz** [comment]
> 试试这个方法：就我而言，当我尝试无线时，它会显示出来。我通过禁用 Windows 防火墙并每次刷新服务器来解决它。我不知道为什么每次打开它都需要刷新，但它对我有用 启动服务器后，转到浏览器并输入完整地址，例如“htttp://192.168.0.104:8080/re”，您的地址可能会有所不同，只需在服务器地址末尾添加“/re”

I can't use this method either.

**Comment by Ra2-IFV** [comment]
TL;DR
Try http://`SERVER_IP`:`SERVER_PORT`/re to refresh, `IP` is `127.0.0.1` if you are accessing this URL from your computer that runs the server; Or use the printed address on server startup. Port is printed out too, usually is `8080`
Just does not work in my case, fuck apple

**Comment by Jim104137** [comment]
will I have the same trouble,I try the way and don't solve that.  my iphone is iphone 15 pro max ios is 17.3.1

> TL;DR如果您從執行伺服器的電腦存取此 URL， 請嘗試使用 http:// `SERVER_IP`: `SERVER_PORT`/re 進行刷新；或在伺服器啟動時使用列印的位址。連接埠也會列印出來，通常是 哦順便說一句，如果您啟用了防火牆，請確保允許此連接埠 只是在我的情況下不起作用，請操蘋果`IP``127.0.0.1``8080`

**Comment by Ra2-IFV** [comment]
> will,I have the same trouble,I try the way and don't solve that. my iphone is iphone 15 pro max ios is 17.3.1
> 
> > TL;DR如果您從執行伺服器的電腦存取此 URL， 請嘗試使用 http:// `SERVER_IP`: `SERVER_PORT`/re 進行刷新；或在伺服器啟動時使用列印的位址。連接埠也會列印出來，通常是 哦順便說一句，如果您啟用了防火牆，請確保允許此連接埠 只是在我的情況下不起作用，請操蘋果`` IP``127.0.0.1``8080 ``

iOS version does not matter
You run sidejitserver on windows or what?

**Comment by Jim104137** [comment]
> > will,I have the same trouble,I try the way and don't solve that. my iphone is iphone 15 pro max ios is 17.3.1
> > > TL;DR如果您從執行伺服器的電腦存取此 URL， 請嘗試使用 http:// `SERVER_IP`: `SERVER_PORT`/re 進行刷新；或在伺服器啟動時使用列印的位址。連接埠也會列印出來，通常是 哦順便說一句，如果您啟用了防火牆，請確保允許此連接埠 只是在我的情況下不起作用，請操蘋果` IP`127.0.0.1`8080 `
> 
> iOS version does not matter You run sidejitserver on windows or what?

I instill in windows 11

**Comment by Dieskz** [comment]
![IMG_3159](https://github.com/user-attachments/assets/7a376d43-e2e7-441c-bdb7-b8ca22d75e7d)

**Comment by Jim104137** [comment]
？

**Comment by Eclipse222** [comment]
@Jim104137 @Ra2-IFV @Dieskz 

Plug your device into iTunes and ensure "Sync with this device over Wi-Fi" is enabled.

Then, run SideJITServer --pair

Then, you can run SideJITServer normally over a network

I hope this helps

**Comment by Jim104137** [comment]
> @Jim104137 @Ra2-IFV @Dieskz 
> 
> 
> 
> Plug your device into iTunes and ensure "Sync with this device over Wi-Fi" is enabled.
> 
> 
> 
> Then, run SideJITServer --pair
> 
> 
> 
> Then, you can run SideJITServer normally over a network
> 
> 
> 
> I hope this helps

Well It doesn't work though I am surely open that

**Comment by hmenzagh** [comment]
Having a similar issue, installed with venv, everything works fine, subsequent launches dont find the device

**Comment by Ra2-IFV** [comment]
I would say try linux......
works on my machine ™

**Comment by hmenzagh** [comment]
Tried to containerise it on Asahi linux, but usb drivers crash during paring (cli tool freezes once the phone is plugged in & popup on my phone), paring files created on the mac do not load on my asahi machine either (I dont know if thats a loading or a machine compatibility issue though)
I'm out of options on my side 🤷
Running on mac & enabling while tethered is my only working option so far ..

**Comment by Dieskz** [comment]
I'm wondering if it's a network problem, such as network structure, firewall and so on.

**Comment by Dieskz** [comment]
> 试图在Asahi Linux上将其容器化，但是在配对过程中USB驱动程序崩溃（一旦手机插上电源并在我的手机上弹出，cli工具就会冻结），在Mac上创建的配对文件也没有在我的Asahi机器上加载（我不知道这是加载还是机器兼容性问题）我在我这边🤷没有其他选择在Mac上运行并在联机时启用是我唯一的工作选项，所以远。。

I don't know how to use Linux.

**Comment by Dieskz** [comment]
> 我会说试试 linux......适用于我的机器 ™

I don't know how to use Linux.

**Comment by hmenzagh** [comment]
Network setup is pretty complex at home, but I know it in and out, I dont think it's the source of my issue here
But after all we are talking about network, so ..
